.github/ISSUE_TEMPLATE/exokit-desktop-bug-report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Exokit Desktop Bug report
+about: File an issue to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps or commands ran to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Additional context**
+Add any other context about the problem here.
+
+**System information:**
+ - OS: [e.g. Windows 10]
+ - Exokit Version [e.g. v0.0.512]
+ - Node/other dependencies versions
```

Comment:
What's "other"?

Reply:
Removed it, having "other" is too general for being in a template